api/src/db/models/user_models.py

```diff
@@ -68,6 +68,10 @@ class User(ApiSchemaTable, TimestampMixin):
     profile: Mapped["UserProfile | None"] = relationship(
         "UserProfile", back_populates="user", uselist=False, cascade="all, delete-orphan"
     )
+    user_type_id: Mapped[int | None] = mapped_column(
+        ForeignKey(LkUserType.user_type_id),
+        default=1,
+    )
```

Comment:
I saw on prior versions you had this with the enum, which is what we'd want to do. We don't want  lookup values to be used directly as integers as it becomes very difficult to work with.

Was there an issue you were running into that prevents this from being `user_type: Mapped[UserType | None]` ?

For context, what the lookup column approach does is let us do something like:

```py
db_session.execute(select(Opportunity).where(Opportunity.opportunity_category == OpportunityCategory.DISCRETIONARY))
```
It handles converting enums to the underlying integers seamlessly.